src/containers/boardMembers/BoardMembersContainer.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { bindActionCreators } from 'redux'
+import { connect } from 'react-redux'
+
+import { getBoardOwner, getBoardContributors, addContributorToBoard, removeContributorFromBoard } from './actions'
+import { getAllUsers } from '../users/actions'
+import BoardMembers from '../../components/boardMembers/BoardMembers'
+
+class BoardMembersContainer extends Component {
+  componentWillMount() {
+    this.props.getBoardOwner(this.props.boardId)
+    this.props.getBoardContributors(this.props.boardId)
+    this.props.getAllUsers()
+  }
+
+  render() {
+    if (this.props.user.id && this.props.owner.id) {
```

Comment:
I don't get this if statement. Are you only checking if IDs aren't undefined ?

Reply:
Yeah, I check if the ids exist to be sure that user and owner have been fetched (else we don't need to display now the members content because all is based on those objects).

Reply:
I've improved this checking (and by extension the current user existence checking)